style.css

```diff
@@ -1 +1,194 @@
 /*  Add your styling here */
+body{
+    margin: 0;
+    font-family: Arial, Helvetica, sans-serif;
+    display: grid;
+    grid-template-areas: 
+    "header"
+    "main"
+    "main2"
+    "footer";
+
+}
+a {
+    color: rgb(122, 0, 122);
+}
+
+
+.header1{
+    grid-area: header;
+    background-color: rgb(235, 195, 123);
+    position: fixed;
+    box-shadow: 1px 1px 4px rgba(0,0,0,.2);
+    width: 100%;
+    z-index: 10;
+    height:75px;
+}
+
+.header1 ul{
+    list-style: none; 
+    overflow: hidden;
+}
+
+.header1 ul a{
+    display: block;
+    /* outline: 1px solid purple; */
+    padding:15px;
+    border:1px solid purple;
+    text-decoration: none;
+}
+
+.header1 .logo{
+    float: left;
+    display: block;
+    margin-top: 5px;
+    margin-left: 5px;
+}
+.header1 ul a:hover{
+    background-color:rgb(226, 165, 52);
+    text-decoration: none;
+}
+.header1 .menu{
+    clear: both;
```

Comment:
If you remove floats, you won't need this rule anymore.

Reply:
I removed it.